package.json

```diff
@@ -96,6 +97,7 @@
     "react-popper": "2.3.0",
     "react-scrollchor": "7.0.2",
     "react-select": "5.5.9",
+    "react-sortable-hoc": "2.0.0",
```

Comment:
It seems the use case for this lib was already covered by `react-dnd`, even though `react-sortable-hoc` seems to have a more straightforward API. Having two libs to achieve the same thing means more maintenance work and a lack of clarity for developers about which ones they should use.

If you think `react-sortable-hoc` is superior, could you open a follow-up issue about migrating all the other places (`AdminContributeCardsContainer`, `SocialLinks` and `EditCollectivePage`) to this new lib?

Reply:
Good point. Used this since it was easy to integrate with React-Select with an existing example, but agree that having two libraries for the same purpose is not great.

Had a look to compare what seems to be the better option overall, and found out that `react-sortable-hoc` is deprecated, so not a great option.

Tried adding `react-dnd` instead, but quickly found issues in making it work smoothly (apart from a more complicated API), such as items sometimes jumping back before they finally land on the correct destination, which is also an issue in the current places where it is being used.

Had a look at the library that `react-sortable-hoc` is being replaced by:[dnd-kit](https://github.com/clauderic/dnd-kit), and this seems like a better option with a simpler API and also seems more future-proof. 

The React-Select example is being updated to use `dnd-kit` as well: https://github.com/JedWatson/react-select/pull/5212

Added it in this PR and will open an issue to also migrate the existing uses of `react-dnd`.

Reply:
Full replacement of `react-dnd` is in this PR: https://github.com/opencollective/opencollective-frontend/pull/8633